src/codegen.rs

```diff
@@ -115,6 +130,32 @@ fn je(n: i8) -> [u8; 2] {
     [0x74, n.to_le_bytes()[0]]
 }
 
+fn call(n: i8) -> [u8; 5] {
```

Comment:
```suggestion
fn call(n: i32) -> [u8; 5] {
```

[`i8::to_le_bytes(self)`](https://doc.rust-lang.org/std/primitive.i8.html#method.to_le_bytes)は`[u8; 1]`を返してきます